lldb/source/Plugins/SymbolFile/DWARF/DWARFUnit.h

```diff
@@ -241,7 +241,7 @@ class DWARFUnit : public UserID {
   FileSpec GetFile(size_t file_idx);
   FileSpec::Style GetPathStyle();
 
-  SymbolFileDWARFDwo *GetDwoSymbolFile();
+  SymbolFileDWARFDwo *GetDwoSymbolFile(bool load_if_needed = true);
```

Comment:
Instead of having a bool here, you could make this an enum with 2 values. Something like this:
```
enum LoadMode : bool { eDoNotForceLoad, eForceLoad };
```